### PR TITLE
etcdctl/ctlv3: protobuf write-out for member list

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/5297.

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @heyitsanthony @xiang90 

Thanks.

**Comment by xiang90** [comment]
for the output documentation, we should explicitly say that some commands do not support protobuf format.

**Comment by gyuho** [comment]
Right, I will update the docs too.

**Comment by gyuho** [comment]
@xiang90 PTAL. Documented those two commands do not support protobuf output.

File: etcdctl/ctlv3/ctl.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379", "127.0.0.1:22379", "127.0.0.1:32379"}, "gRPC endpoints")
 
-	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, protobuf, table)")
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, protobuf, table). Some commands do not support table/protobuf format. Please check README for more detail.")
```
Comment (gyuho) [inline etcdctl/ctlv3/ctl.go]: @xiang90 Added this comment. PTAL. Thanks.

**Comment by gyuho** [comment]
/cc @xiang90 PTAL.

Thanks.

File: etcdctl/ctlv3/ctl.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379", "127.0.0.1:22379", "127.0.0.1:32379"}, "gRPC endpoints")
 
-	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, protobuf, table)")
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, etc). Check README for more options.")
```
Comment (xiang90) [inline etcdctl/ctlv3/ctl.go]: set the output format (simple, json, etc..)

File: etcdctl/READMEv3.md [hunk 3]
```diff
@@ -484,6 +484,10 @@ On success, prints a humanized table of each endpoint URL, ID, version, database
 
 On success, prints a line of JSON encoding each endpoint URL, ID, version, database size, leadership status, raft term, and raft status. On failure, returns with a non-zero exit code.
 
+##### Protobuf reply
```
Comment (xiang90) [inline etcdctl/READMEv3.md]: we should document which command supports protobuf and links to the protobuf definition.

Comment (gyuho) [inline etcdctl/READMEv3.md]: Just added that doc for `member list`. Other commands with protobuf are already documented with the links.

**Comment by xiang90** [comment]
LGTM

File: etcdctl/ctlv3/ctl.go [hunk 4]
```diff
@@ -45,7 +45,7 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379", "127.0.0.1:22379", "127.0.0.1:32379"}, "gRPC endpoints")
 
-	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, protobuf, table)")
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, etc..). Check README for more options.")
```
Comment (xiang90) [inline etcdctl/ctlv3/ctl.go]: remove the mention of readme file.

Comment (xiang90) [inline etcdctl/ctlv3/ctl.go]: when users run etcdctl, they have no idea what README is.

**Comment by gyuho** [comment]
Just removed. Merging in greens. Thanks!

**Pull request merged**